src/AddTodoForm.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'; 
+
+function AddTodoForm({ onAddTodo }) {
+    const [todoTitle, setTodoTitle] = useState('');
+
+  function handleTitleChange(event){
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  }
+  function handleAddTodo(event){
+    event.preventDefault();
+    onAddTodo(
+      {
+        title: todoTitle,
+        id: Date.now()
+      }
+    )
+
+    setTodoTitle("");
+  }
+  return (
+    <form onSubmit={(event) => handleAddTodo(event)}>
+      <label htmlFor="todoTitle">Title </label>
+      <input type="text" name="title" value={todoTitle}onChange={(event) => handleTitleChange(event)} />
```

Comment:
nit: add space between `value={todoTitle}` and `onChange={(ev...`

Reply:
btw, nits are what I call small changes or tweaks

Reply:
Thanks for highlighting that. Looks like my Prettier extension isn't working. Is there a code formatter you recommend? It looks like spacing was corrected on lesson-1-6 and it isn't an issue on current branch.